### DISCUSSION TITLE
How can I contribute my translation

### DISCUSSION THREAD

**Discussion by HfSrJun** [IntelligenceModding/AdvancedPeripherals#665] [Q&A]
I translated this mod completely to zh_cn almost 1 year ago (PR #472) but was rejected because the translation is maintained via crowdin.
When I opened crowdin I saw nothing, so I thought the crowdin project is unfinished. I forgot this thing due to real life bussiness and never contribute again.
Not long ago I noticed the zh_cn translation was added, but the quality is not good enough, and the manual is not included. After search I found the contributer just in my PR, and the crowdin project is broken somehow.
So I think it is better to upload my manual, and improve the translation for lang file.
In short, can anyone fix the crowdin project?
